pyqtgraph/__init__.py

```diff
@@ -463,3 +464,14 @@ def stack(*args, **kwds):
     except NameError:
         consoles = [c]
     return c
+
+
+def applyPalette(app,name):
+    name = name.lower()
+    if name == "light":
```

Comment:
I would like to advertise (in the docs) a name that is specific to QDarkStyle here.
'qdarkstyle' and 'qdarkstylelight' come to mind.
We can default 'dark' and 'light' to these, but we might point these simple names at palettes that do not depend on a stylesheet in the future?
To clarify
`if name in ['qdarkstylelight', 'light']:`
and
`if name in ['qdarkstyle','dark']: `
seem more appealing to me.

The docs will only mention the qdarkstyle names for now.

Reply:
To avoid confusion in the future about why the `'dark'` palette suddenly looks different with the qdarkstyle dark palette stylesheet, should we just stick with names specific to qdarkstyle?
```
if name == "qdarkstylelight":
    ...
elif name =="qdarkstyledark"
    ...
```

Reply:
I think those would be good canonical names.
I would tend towards "qdarkstyle" for the dark style, that seems less redundant.
I wouldn't mind an alias to "qdarkstyledark". 

You are probably right about not having a 'dark' label until we think about that some more.

Reply:
Okay, the changed to:

``` 
        if style == 'qdarkstylelight':
            p = palette.getQDarkStyleLightQPalette()
        elif style in ['qdarkstyle','qdarkstyledark']:
            p = palette.getQDarkStyleDarkQPalette()
```

Reply:
I like that :)